content/asset-tracking/using-the-sdks.textile

```diff
@@ -218,6 +272,12 @@ blang[android].
   ...
   ```
 
+  ```[swift]
+  // Start tracking an asset with its tracking ID
+  publisher?.track(trackable: Trackable(id: trackingId),
+                   constraints: exampleConstraints)
```

Comment:
The question mark is there because the field is optional/ nullable in the class the code came from. This is out of context for this example, we can remove `?`.

Also, this method (`func track(trackable: Trackable, completion: @escaping ResultHandler<Void>)`) takes 2 arguments, trackable and the completion handler. For example, (from `MapViewController.swift`):
```swift
        publisher?.track(trackable: trackable) { [weak self] result in
            switch result {
            case .success:
                self?.trackables = [trackable]
                logger.info("Initial trackable tracked successfully.")
            case .failure(let error):
                self?.locationState = .failed
                self?.refreshAnnotations()
                self?.showErrorDialog(error: error)
            }
        }
```

Reply:
Great, thanks